spec/controllers/course_summaries_controller_spec.rb

```diff
@@ -130,7 +130,7 @@
 
     it 'not be able to CSV graders report' do
       get_as @grader, :download_csv_grades_report
-      expect(response.status).to eq(404)
+      expect(response.status).to eq(403)
```

Comment:
Since you're changing all of these anyway, do you want to switch to the preferred `expect(response).to have_http_status(403)`?